### PR TITLE
[doc] [jsk_pcl_ros_utils] [jsk_pcl_ros] Add documentation

### LINKED ISSUE #1631: [doc] No Documentation Nodes
There is no document about nodes below.
Author or people familiar with node should write doc, or no one reuse it and others duplicate similar node.

```
jsk_pcl/InteractiveCuboidLikelihood
jsk_pcl/FindObjectOnPlane
jsk_pcl/DepthCalibration
jsk_pcl/LineSegmentCollector
jsk_pcl/LineSegmentDetector
jsk_pcl/BilateralFilter
jsk_pcl/JointStateStaticFilter
jsk_pcl/OrganizedPassThrough
jsk_pcl/BoundingBoxFilter
jsk_pcl/MultiPlaneSACSegmentation
jsk_pcl/EdgebasedCubeFinder
jsk_pcl/ParallelEdgeFinder
jsk_pcl/EdgeDepthRefinement
jsk_pcl/OrganizedEdgeDetector
jsk_pcl/HandleEstimator
jsk_pcl/GridSampler
jsk_pcl/EnvironmentPlaneModeling
jsk_pcl/ColorHistogramMatcher
jsk_pcl/NormalEstimationIntegralImage
jsk_pcl/SelectedClusterPublisher
jsk_pcl/ImageRotateNodelet
jsk_pcl/VoxelGridDownsampleManager
jsk_pcl/VoxelGridDownsampleDecoder
jsk_pcl/ColorizeRandomForest
jsk_pcl/ColorizeMapRandomForest
jsk_pcl/PointCloudMoveitFilter

jsk_pcl_utils/TransformPointcloudInBoundingBox
jsk_pcl_utils/DelayPointCloud
jsk_pcl_utils/NormalConcatenater
jsk_pcl_utils/TfTransformBoundingBoxArray
jsk_pcl_utils/ColorizeDistanceFromPlane
jsk_pcl_utils/PolygonArrayWrapper
jsk_pcl_utils/PolygonArrayUnwrapper
jsk_pcl_utils/PolygonAppender
jsk_pcl_utils/StaticPolygonArrayPublisher
jsk_pcl_utils/PlaneRejector
jsk_pcl_utils/PlaneReasoner
```

### PR DESCRIPTION
**Pull request by YutoUchimi**
Closes #1631 

This PR fixes code minimally and adds sample, test and documentation for below nodes.

__jsk_pcl_ros__
- [x] jsk_pcl/InteractiveCuboidLikelihood
- [x] jsk_pcl/FindObjectOnPlane
- [x] jsk_pcl/BilateralFilter
- [x] jsk_pcl/JointStateStaticFilter
- [x] jsk_pcl/MultiPlaneSACSegmentation
- [x] jsk_pcl/EdgebasedCubeFinder
- [x] jsk_pcl/HandleEstimator
- [x] jsk_pcl/EnvironmentPlaneModeling
- [x] jsk_pcl/SelectedClusterPublisher 
- [x] jsk_pcl/VoxelGridDownsampleManager
- [x] jsk_pcl/VoxelGridDownsampleDecoder
- [x] jsk_pcl/ColorizeRandomForest
- [x] jsk_pcl/ColorizeMapRandomForest
- ~jsk_pcl/PointCloudMoveitFilter~ (This is not a node but a MoveIt plugin.)

__jsk_pcl_ros_utils__
- [x] jsk_pcl_utils/TransformPointcloudInBoundingBox
- [x] jsk_pcl_utils/NormalConcatenater
- [x] jsk_pcl_utils/PolygonAppender
- [x] jsk_pcl_utils/StaticPolygonArrayPublisher
- [x] jsk_pcl_utils/PlaneRejector
- [x] jsk_pcl_utils/PlaneReasoner

### CODE REVIEW

**Comment by YutoUchimi** [comment]
Done.
@knorth55 Colud you please review this when you have time?

BTW, I think we have to re-enable build trigger for jsk_docs in Read the Docs (https://github.com/jsk-ros-pkg/jsk_docs/issues/25) before or after this PR is merged.

File: doc/jsk_pcl_ros/nodes/environment_plane_modeling.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+These 5 topics described above must be synchronized.
+
+* `~input/leg_bounding_box` (`jsk_recognition_msgs/BoundingBox`)
+
+  Sorry, editing now...
+
+* `/move_base_simple/goal` (`geometry_msgs/PoseStamped`)
+
+  Sorry, editing now...
```
Comment (knorth55) [inline doc/jsk_pcl_ros/nodes/environment_plane_modeling.md]: is this WIP?

File: doc/jsk_pcl_ros/nodes/environment_plane_modeling.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+* `~input/coefficients` (`jsk_recognition_msgs/ModelCoefficientsArray`)
+
+  Input normal coefficients of plane.
+
+These 5 topics described above must be synchronized.
+
+* `~input/leg_bounding_box` (`jsk_recognition_msgs/BoundingBox`)
+
+  Sorry, editing now...
```
Comment (knorth55) [inline doc/jsk_pcl_ros/nodes/environment_plane_modeling.md]: is this WIP?

File: doc/jsk_pcl_ros/nodes/environment_plane_modeling.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+  Occupancy grid of plane before applying morphological dilation & erosion.
+
+* `~debug/noeroded_grid_map` (`jsk_recognition_msgs/SimpleOccupancyGridArray`)
+
+  Occupancy grid of plane to which morphological filter is applied, but not enough erosion.
+
+* `/footstep_simple/goal` (`geometry_msgs/PoseStamped`)
+
+  Sorry, editing now...
```
Comment (knorth55) [inline doc/jsk_pcl_ros/nodes/environment_plane_modeling.md]: is this WIP?

File: doc/jsk_pcl_ros/nodes/environment_plane_modeling.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+
+## Parameter
+
+* `~complete_footprint_region` (Bool, default: `False`)
+
+  Sorry, editing now...
```
Comment (knorth55) [inline doc/jsk_pcl_ros/nodes/environment_plane_modeling.md]: is this WIP?

File: doc/jsk_pcl_ros/nodes/environment_plane_modeling.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+* `~footprint_frames` (List of String, default: `[]`)
+
+  Sorry, editing now...
```
Comment (knorth55) [inline doc/jsk_pcl_ros/nodes/environment_plane_modeling.md]: is this WIP?

**Comment by knorth55** [comment]
I ran all tests and they passed in my environment

File: doc/jsk_pcl_ros/nodes/environment_plane_modeling.md (hunk 5 above)
Comment (YutoUchimi) [reply doc/jsk_pcl_ros/nodes/environment_plane_modeling.md]: Thanks for review. I updated the doc.

**Comment by knorth55** [comment]
I confirmed that all tests are enabled in CMakelists.txt

**Comment by k-okada** [comment]
Great work!!

**Pull request merged**